.travis.yml

```diff
@@ -1,13 +1,14 @@
 language: scala
 scala:
-  - 2.11.7
+  - 2.11.8
+  - 2.10.6
 jdk:
   - oraclejdk8
 
 script:
-  - sbt +clean coverage +test coverageReport
+  - sbt clean coverage test coverageReport
 after_success:
-  - sbt coveralls
+  - sbt 'project core_1-12' coveralls
```

Comment:
This hack allows me to only upload the coverage for the `core_1.12` project. This is because it is the only project that generates reports, and if only one project has a report than `coverageAggregate` refuses to aggregate the single report and `sbt coveralls` fails to find it in the subdirectory.